docs: refactor desktop access intro (#42110)
gravitational/teleport: docs/pages/desktop-access/introduction.mdx

videoBanner: n2h0GisWdss
---

The topics in this guide describe how to configure Teleport to provide secure, passwordless
access to Microsoft Windows desktops and servers. For Windows, Teleport provides the
following key features:

- Passwordless access to Windows hosts backed by secure cryptographic
  authentication and short-lived certificates.
- Configurable role-based access controls (RBAC) for groups of hosts and users.
- Configurable clipboard and directory sharing for copying and pasting to and
  from remote Windows hosts.
- Session recording for all desktop activity.
- Audit logs that track user activity.

## How it works

In order to provide connectivity to Windows hosts, you deploy one or more
instances of Teleport's Windows Desktop Service on your infrastructure. The
Windows Desktop Service typically makes an outbound connection to the Teleport
Proxy Service and maintains a reverse tunnel that is used to power remote

> desktop sessions. Since the only networking requirement is a single outbound

Reviewer: ```suggestion
desktop sessions. 

Since the only networking requirement is a single outbound
```

I found the paragraph a little long.

> connection to the Teleport Proxy, you are free to deploy as many desktop

Reviewer: ```suggestion
connection to the Teleport Proxy Service, you are free to deploy as many desktop
```

> connection to the Teleport Proxy, you are free to deploy as many desktop
> services in as many different network segments as you need - Teleport will take

Reviewer: ```suggestion
connection to the Teleport Proxy, you are free to deploy as many Desktop
Service instances in as many different network segments as you need - Teleport will take
```

> care of selecting a route to your Windows host at connection time.

Reviewer 2: > Since the only networking requirement is a single outbound connection to the Teleport Proxy

There's also a networking requirement of wds --> RDP server.

> ![Manage access to Windows through Teleport](../../img/desktop-access/windows-desktop-service-overview.png)

Author: I'd also like to remove/replace this diagram, but don't have the time to do so right now and I think the rest of the changes here are valuable even without updating the diagram.

You can configure Teleport Windows Desktop Service to control access for the following
scenarios:

- Domain users who access computers that are joined to an Active Directory domain.
- Local users who access computers that aren't joined to an Active Directory domain.

In both cases, Teleport authenticates by presenting a short lived certificate to
the RDP server in the form of a virtual smart card - no passwords are involved.

Each approach offers its own pros and cons.

Teleport's support for domain users works with standard Active Directory smart
card support, which comes with several benefits:

- automatic inventory management via an LDAP integration

> - congfiguration can be managed centrally via group policy

Reviewer: ```suggestion
- configuration can be managed centrally via Group Policy
```

- no Teleport software runs on the Windows hosts

The downside of the Active Directory setup is that Windows PKI can be difficult
to configure and troubleshoot, and some environments require large configuration
changes in order to support smart card authentication.

Teleport's support for local users requires the installation of a separate
package on each host that you wish to provide access to. System configuration is

> performed by the installer rather than via group policy. This approach is easier

Reviewer 2: I think Group Policy is treated as a proper noun e.g. https://learn.microsoft.com/en-us/previous-versions/windows/it-pro/windows-server-2012-r2-and-2012/hh831791(v=ws.11)

```suggestion
performed by the installer rather than via Group Policy. This approach is easier
```

to set up and easier to troubleshoot, and also provides unique features like the
ability to create users on-demand. The lack of automatic LDAP-based discovery
and the requirement to install software on each host makes this approach less

> attractive for large environments.

Reviewer 3: Do we want to mention somewhere that you can automate it using installer in CLI with something like Terraform?
